geodiff/src/geodiff.cpp

```diff
@@ -74,7 +74,7 @@ int GEODIFF_createChangeset( const char *base, const char *modified, const char
     db->open( modified );
 
     Buffer sqlBuf;
-    sqlBuf.printf( "ATTACH '%s' AS aux", base );
+    sqlBuf.checkStmtAndPrintf( "ATTACH '%s' AS aux", base );
```

Comment:
could we use `%q` instead of `%s` here? It seems that sqlite's printf should handle escaping of quotes there: https://www.sqlite.org/printf.html#percentq

Reply:
Great, this solution is way better, thanks! Do we want to use it everywhere instead of `%s`?

Reply:
yes, it would be good to check if there are any other such cases while at it :-)